ci/release.groovy

```diff
@@ -2,22 +2,53 @@
 
 pipeline {
   agent none
+   environment {
+    GITHUB_TOKEN = credentials('gh-token-mesosphere-ci-dcos-deploy')
+    APPLE_DEVACC = credentials('TEMP_APPLE_DEVELOPER_ACCOUNT')
+    GOLANG_VER = "1.13.7"
+    GON_VER = "0.2.2"
+  }
 
   options {
     timeout(time: 6, unit: 'HOURS')
   }
 
   stages {
     stage("Build binaries") {
-      agent { label 'mesos-ubuntu' }
+      agent {
+        label "mac"
+      }
 
       steps {
+        ansiColor('xterm') {
+          sh '''
+              security delete-keychain jenkins-${JOB_NAME} || :
+              security create-keychain -p test jenkins-${JOB_NAME}
+              security unlock-keychain -p test jenkins-${JOB_NAME}
+              security list-keychains -d user -s jenkins-${JOB_NAME}
+              security default-keychain -s jenkins-${JOB_NAME}
+              cat ${SIGNING_CERTIFICATE} > cert.p12
+              security import cert.p12 -k jenkins-${JOB_NAME} -P ${SIGNING_CERTIFICATE_PASSWORD} -T /usr/bin/codesign
+              security set-key-partition-list -S apple-tool:,apple:,codesign: -s -k test jenkins-${JOB_NAME}
+              xcrun altool --notarization-history 0 -u "${APPLE_DEVACC_USR}" -p "${APPLE_DEVACC_PSW}"
+            '''
+          sh 'security find-identity -v | grep -q 5C99CF2B12B186C69A37BD21DFF24ACBE10560CF'
```

Comment:
What's `5C99CF2B12B186C69A37BD21DFF24ACBE10560CF`? Can we set a more descriptive variable like we did for `GON_VER`?

Reply:
it is the identity ID, but sure it can be set as environment variable at the beginning of the Jenkinsfile

Reply:
Sure, this might add to the readability, will add this as the variable ID_HASH